src/lib/core.js

```diff
@@ -13,7 +13,7 @@ const Schema = Immutable.Record({
   'cobiJsToken': null,
   'panel': 'simulator',
   'containerUrl': null,
-  'thumbControllerType': 'COBI',
+  'thumbControllerType': 'COBI.bike',
```

Comment:
@simonseyer please dont change this. This comes from the Spec not from the code here. Unless we also change the TC name in the spec I would prefer to keep all names consistent.